src/core_plugins/kibana/public/home/components/tutorial/tutorial.test.js

```diff
@@ -97,13 +97,13 @@ describe('isCloudEnabled is false', () => {
       tutorialId={'my_testing_tutorial'}
       bulkCreate={() => {}}
     />);
-    loadBasicTutorialPromise.then(() => {
-      component.update();
-      expect(component).toMatchSnapshot(); // eslint-disable-line
-    });
+
+    await loadBasicTutorialPromise;
+    component.update();
+    expect(component).toMatchSnapshot(); // eslint-disable-line
   });
 
-  test('should display ON_PREM_ELASTIC_CLOUD instructions when toggle is clicked', () => {
+  test.skip('should display ON_PREM_ELASTIC_CLOUD instructions when toggle is clicked', async () => {
```

Comment:
Are the changes in here relevant to this PR, including skipping the test?

Reply:
They are relevant, there were some subtle differences in how react-test-utils works. In order to get x-pack and kibana root on the same version I had to make a couple tweaks to tests. I did miss this skipped one! Will fix.

Reply:
Ah now I remember better, these tests weren't actually doing anything since they weren't async + didn't have a done callback. So the tests were silently failing this whole time because they asserted after promise callbacks, leading to UnhandlesPromiseRejection exceptions that didn't fail the test.